client/extensions/woocommerce/components/reading-widget/index.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import emailValidator from 'email-validator';
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+import ExternalLink from 'components/external-link';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FormTextInput from 'components/forms/form-text-input';
+import { getSitePostsForQueryIgnoringPage } from 'state/posts/selectors';
+import humanDate from 'lib/human-date';
+import MultiCheckbox from 'components/forms/multi-checkbox';
+import QueryPosts from 'components/data/query-posts';
+
+class ReadingWidget extends Component {
+
+	state = {
+		emailAddress: '',
+		roles: [ 'storeOwner' ],
+		subscriptionFormExpanded: false,
+	};
+
+	static propTypes = {
+		query: PropTypes.object,
+		siteId: PropTypes.number,
+		text: PropTypes.string,
+		title: PropTypes.string,
+	};
+
+	static defaultProps = {
+		query: { category: 'start-your-store', number: 3 },
+		siteId: 113771570,
```

Comment:
Agreed. Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc